src/web/lib/hakuneko/frontend@modern/about.md

```diff
@@ -0,0 +1,67 @@
+# Modern Theme
```

Comment:
What is the purpose of this file?
I recommend to track TODOs as GitHub issues and changes are traceable via the commit history in git.